client-src/elements/chromedash-admin-blink-page.ts

```diff
@@ -203,7 +202,7 @@ export class ChromedashAdminBlinkPage extends LitElement {
           (component, index) => html`
             <li class="layout horizontal" id="${component.name}">
               <chromedash-admin-blink-component-listing
-                .id=${component.id}
+                .id=${component.componentId}
```

Comment:
When you change the name of an attribute or property in an element, you have to update the attributes and property assignments in uses of that element. You also need to make sure that when you access a field on an object, that field actually exists: `component` is an `OwnersAndSubscribersOfComponent`, which has an `id` field and not a `componentId` field. So this needs to change to

```suggestion
                .componentId=${component.id}
```

You also need to find a way to test this and any other uses of the property, to be sure you've correctly understood what needs to match. @jrobbins, what's a good way for Mark to test this?